src/witan/sen2/return/person_level/blade_export/csv/plans_placements.clj

```diff
@@ -585,3 +602,136 @@
                            :label         "Issue | TOTAL"
                            :summary-label "Summary"
                            :desc          "Index: Issue (summary statistic)"}))))
+
+
+
+
+;;; # CSV file read
+(defn csv-file->ds
+  "Read columns required to construct census
+   from CSV file of `plans-placements-on-census-dates` into a dataset."
+  [filepath]
+  (tc/dataset filepath
+              {:column-whitelist (map name key-columns-for-census)
```

Comment:
allowlist

Reply:
TMD deps bumped to 7.010 (TC still at 7.007) and anachronistic keywords updated.